supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/PreviousRetryReasons.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.backend.impl.jes
+
+import cromwell.services.keyvalue.KeyValueServiceActor._
+import lenthall.validation.ErrorOr.ErrorOr
+import cats.syntax.validated._
+import cats.syntax.cartesian._
+
+import scala.util.{Failure, Success, Try}
+import JesBackendLifecycleActorFactory.preemptionCountKey
+import JesBackendLifecycleActorFactory.unexpectedRetryCountKey
+
+case class PreviousRetryReasons(preempted: Int, unexpectedRetry: Int) {
+  // E.g. preempted=3, unexpectedRetry=4, maxAttempts=7 => false
+  // E.g. preempted=3, unexpectedRetry=4, maxAttempts=8 => true
+  def retryable(maxAttempts: Int) = {
+
+  }
```

Comment:
Not sure where this was used in the past, but this dead code returns `Unit` and doesn't have any callers.